component/vshn_postgres.jsonnet

```diff
@@ -54,7 +58,65 @@ local composition =
                         },
                       },
                     };
-
+  local localca = comp.KubeObject('cert-manager.io/v1', 'Issuer') +
+                  {
+                    spec+: {
+                      forProvider+: {
+                        manifest+: {
+                          metadata: {
+                            name: '',
+                            namespace: '',
+                          },
+                          spec: {
+                            selfSigned: {
+                              crlDistributionPoints: [],
+                            },
+                          },
+                        },
+                      },
+                    },
+                  };
+  local certificate = comp.KubeObject('cert-manager.io/v1', 'Certificate') +
+                      {
+                        spec+: {
+                          forProvider+: {
+                            manifest+: {
+                              metadata: {
+                                name: '',
+                                namespace: '',
+                              },
+                              spec: {
+                                secretName: certificateSecretName,
+                                duration: '87600h',
+                                renewBefore: '2400h',
+                                subject: {
+                                  organizations: [
+                                    'vshn-appcat',
+                                  ],
+                                },
+                                isCA: false,
+                                privateKey: {
+                                  algorithm: 'RSA',
+                                  encoding: 'PKCS1',
+                                  size: 2048,
```

Comment:
I'd probably go with 4096 instead of 2048 for RSA.

Reply:
ok

Reply:
fixed